service.yml

```diff
@@ -0,0 +1,28 @@
+name: kafka-images
+lang: unknown
+lang_version: unknown
+git:
+  enable: true
+codeowners:
+  enable: true
+semaphore:
+  enable: true
+  pipeline_type: cp-dockerfile
+  docker_repos: [
+    'unsynced-confluentinc/cp-server-connect',
```

Comment:
there was an issue with using unsynced (esp for images used as based images). Can you convert these back to `confluentinc/*` for now? Ill need to update the documentation.